src/modules/Mail.rb

```diff
@@ -216,6 +216,15 @@ def ProbePackages
           )
         end
 
+        if !Package.Installed("spamassassin")
+          #If spamassassin is not installaed amavisd can not work
```

Comment:
Well, then amavisd-new.rpm must Require spamassassin.rpm, YaST is not the place to add such dependencies. And amavisd-new *does* have such a dependency already. This part of the PR should not be needed.